Makefile

```diff
@@ -77,8 +77,8 @@ test-run: build bin/minio bin/$(ZEEKPATH) bin/$(SURICATAPATH)
 test-heavy: build $(SAMPLEDATA)
 	@go test -v -tags=heavy ./tests
 
-test-zeek: bin/$(ZEEKPATH)
-	@ZEEK=$(CURDIR)/bin/$(ZEEKPATH)/zeekrunner go test -v -run=PcapPost -tags=zeek ./zqd
+test-pcap-ingest: bin/$(ZEEKPATH)
+	@ZEEK=$(CURDIR)/bin/$(ZEEKPATH)/zeekrunner go test -v -run=PcapPost -tags=pcapingest ./zqd
```

Comment:
Maybe use test-pcappost as the makefile goal and pcappost as the Go build tag for symmetry with the test name?